srv/salt/ceph/mon/key/default.sls

```diff
@@ -13,8 +11,9 @@
     - makedirs: True
     - context:
       mon_secret: {{ salt['keyring.secret'](keyring_file) }}
-      admin_secret: {{ salt['keyring.secret'](admin_keyring) }}
     - fire_event: True
 
-
-
+{{ keyring_file }} append admin keyring:
+  file.append:
```

Comment:
yes salt takes care of that.
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.append